netlify.toml

```diff
@@ -0,0 +1,8 @@
+# https://docs.netlify.com/configure-builds/file-based-configuration/
+
+[build]
+publish = "public/"
+command = "yarn build"
```

Comment:
Should these be indented? (likewise for line 8)

Reply:
As discussed, I'd rather not indent.